src/Domain/Creators/Shell.hpp

```diff
@@ -91,10 +90,20 @@ class Shell : public DomainCreator<3> {
         "The equatorial compression factor."};
   };
 
-  struct UseLogarithmicMap {
-    using type = bool;
+  struct RadialPartitioning {
+    using type = std::vector<double>;
+    static constexpr Options::String help = {
```

Comment:
Either in doxygen or the help string for this option, please add some text that encourages the user to select as few partitions as possible to minimize the number of Blocks and maximize  the amount of potential h-refinement